eclass/xorg-3.eclass

```diff
@@ -129,6 +146,36 @@ BDEPEND+=" ${EAUTORECONF_DEPENDS}"
 unset EAUTORECONF_DEPENDS
 unset EAUTORECONF_DEPEND
 
+if [[ ${FONT} == yes ]]; then
+	RDEPEND+=" media-fonts/encodings
+		>=x11-apps/mkfontscale-1.2.0"
+	PDEPEND+=" media-fonts/font-alias"
+	DEPEND+=" >=media-fonts/font-util-1.2.0
+		>=x11-apps/mkfontscale-1.2.0"
```

Comment:
I wonder if any of these should be `BDEPEND`? I suspect the answer is no, since e.g. `mkfontscale` is in RDEPEND so it can be used in `pkg_post_rm`, etc, but I'm not 100% sure.

Just sort of thinking out loud. I don't expect anything needs to change here.